tools/c7n_salactus/c7n_salactus/inventory.py

```diff
@@ -62,7 +62,18 @@ def load_manifest_file(client, bucket, schema, versioned, ifilters, key_info):
             yield keys
 
 
+def is_delete_marker(ischema, kr):
+    """Is the given key record for a delete marker?
+        "fileSchema" : "Bucket, Key, VersionId, IsLatest,
+                        IsDeleteMarker, Size, LastModifiedDate, EncryptionStatus"
+    """
+    return kr[4] == 'true'
+
+
 def inventory_filter(ifilters, ischema, kr):
+    if is_delete_marker(ischema, kr):
```

Comment:
part of the earlier review.. at this scale (aka billions), function calls have overhead, rather than doing a functional call for a one liner, inlining this with comment seems more appropriate.

Reply:
also for a non versioned bucket is the schema the same...

Reply:
Correct - Will need to revisit the other hard-coded indexes.

Reply:
ischema can be used to do some of this checking conditionally.